src/reference-guide/migration.md

```diff
@@ -0,0 +1,96 @@
+# Migrating from Deis v1
+
+Workflow uses [`kubectl`][kubectl] and [`helmc`][helmc] to manage the cluster. These tools
+are equivalent to Deis v1's [`fleetctl`][fleetctl] and [`deisctl`][deisctl]. These two tools are
+used for managing the cluster's state, installing the platform and inspecting its state.
+
+This document is a "cheat cheet" for users migrating from Deis v1 to Workflow (v2). It lists most of
+the known commands administrators would use with `deisctl` and translates their usage in Workflow.
+
+## Listing all Components
+
+```
+# Deis v1
+$ deisctl list
+$ fleetctl list-units
+
+# Workflow
+$ kubectl --namespace=deis get pods
+$ kubectl get pods --all-namespaces
```

Comment:
Same comment as above, since this is the analog to the other command I commented on.